languages/SMW_LanguageAr.php

```diff
@@ -52,62 +54,58 @@ class SMWLanguageAr extends SMWLanguage {
 
 	protected $m_SpecialProperties = array(
 		// always start upper-case
-		'_TYPE'  => 'لديه نوع',
-		'_URI'   => 'معرف الموارد الموحد معادلة',
-		'_SUBP' => 'الخاصية الفرعية ل',
-		'_SUBC' => 'تصنيف فرعي من',
-		'_UNIT' => 'عرض الوحدات',
-		'_IMPO' => 'المستوردة من',
-		'_CONV' => 'يقابل',
-		'_SERV' => 'يوفر الخدمة',
-		'_PVAL' => 'يسمح بالقيمة',
+		'_TYPE' => 'من النوع',
+		'_URI'  => 'URL مكافئ',
+		'_SUBP' => 'خصيصة فرعية من',
+		'_SUBC' => 'مُصنّف على التصنيف',
+		'_UNIT' => 'وحدة العرض',
+		'_IMPO' => 'مستوردة من',
+		'_CONV' => 'توافق',
+		'_SERV' => 'تقدّم خدمة',
+		'_PVAL' => 'تقبل القيمة',
 		'_MDAT' => 'تاريخ التعديل',
-		'_CDAT' => 'Creation date', // TODO: translate
-		'_NEWP' => 'Is a new page', // TODO: translate
-		'_LEDT' => 'Last editor is', // TODO: translate
-		'_ERRP' => 'يمتلك قيمة غير صحيحة ل',
-		'_LIST' => 'يمتلك حقول',
-		'_SOBJ' => 'Has subobject', // TODO: translate
-		'_ASK'  => 'Has query', // TODO: translate
-		'_ASKST'=> 'Query string', // TODO: translate
-		'_ASKFO'=> 'Query format', // TODO: translate
-		'_ASKSI'=> 'Query size', // TODO: translate
-		'_ASKDE'=> 'Query depth', // TODO: translate
-		'_ASKDU'=> 'Query duration', // TODO: translate
-		'_MEDIA'=> 'Media type',
-		'_MIME' => 'MIME type',
-		'_ERRC' => 'Has processing error',
-		'_ERRT' => 'Has processing error text',
-		'_PREC'  => 'Display precision of',
-		'_LCODE' => 'Language code',
-		'_TEXT'  => 'Text',
-		'_PDESC' => 'Has property description',
-		'_PVAP'  => 'Allows pattern',
-		'_DTITLE' => 'Display title of',
-		'_PVUC' => 'Has uniqueness constraint',
+		'_CDAT' => 'تاريخ الإنشاء',
+		'_NEWP' => 'هي صفحة جديدة',
+		'_LEDT' => 'آخر مَن حرّرها',
+		'_ERRP' => 'فيها قيمة غير صحيحة في',//الخصيصة
```

Comment:
> The issue of using diacritics is rather controversial on Arabic Wikipedia.

I do not think that we should try to solve a problem here which exists on Arabic Wikipedia for 15 years - obviously without conclusion.

> Anyway, the Arabic MediaWiki translations file mostly doesn't contain diacritics.

So that's what we should do here, too. Note at all to avoid misunderstandings: I am not judging whether diacritics are better to use or not.

@Meno25 The reason I asked for a second opinion was just to make sure that the translations as such are sensible and understandable in the light of their context. This seems to be the case.